repos/fdbt-site/cypress_tests/cypress/support/helpers.ts

```diff
@@ -610,3 +583,16 @@ export const clearDates = (): void => {
     getElementById('end-month-input').clear();
     getElementById('end-year-input').clear();
 };
+
+export const completeMultiServicePages = (): void => {
+    
+    cy.get('[class=govuk-checkboxes__item]').each((checkbox) => {
+        cy.wrap(checkbox).click();
+    });
+    
+    getElementById('operator-1').click();
+    cy.get('[class=govuk-checkboxes__item]').each((checkbox) => {
+        cy.wrap(checkbox).click();
```

Comment:
possible extension randomly clicking checkboxes rather than clicking all or clicking select all button?

Reply:
it is the same as we do in clickAllCheckboxes() in helpers.ts